jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/complex/menu/BasicMenuTests.java

```diff
@@ -0,0 +1,59 @@
+package io.github.epam.angular.tests.elements.complex.menu;
+
+import io.github.epam.TestsInit;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+import static com.epam.jdi.light.angular.elements.common.NestedDropdownMenu.BASIC_MENU_VALUES;
+import static io.github.com.StaticSite.angularPage;
+import static io.github.com.pages.sections.MenuSection.basicMenu;
+import static io.github.com.pages.sections.MenuSection.basicMenuSelectedOption;
+import static io.github.epam.site.steps.States.shouldBeLoggedIn;
+
+public class BasicMenuTests extends TestsInit {
+
+    @BeforeTest
```

Comment:
This method does not perform as expected. The attribute (alwaysRun = true) is missing here. Also I think it should be @BeforeMethod annotation and not @BeforeTest. Maybe this will fix test failures.

Reply:
Fixed, thanks.